modules/ztp-image-based-upgrade-procedure-one-step.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-procedure-one-step_{context}"]
+= Performing an image-based upgrade on managed clusters at scale in one step
+
+For use cases when service interruption is not a concern, you can upgrade a set of your managed clusters by using the `ImageBasedGroupUpgrade` CR with several actions combined in one step with one rollout strategy.
+With one rollout strategy, the upgrade time can be reduced but you can only troubleshoot failed clusters after the upgrade plan is complete.
+
+.Prerequisites
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+* You have created policies and `ConfigMap` objects for resources used in the image-based upgrade.
+* You have installed the {lcao} and OADP Operators on all managed clusters through the hub cluster.
+
+.Procedure
+
+. Create a YAML file on the hub cluster that contains the `ImageBasedGroupUpgrade` CR:
++
+--
+include::snippets/ibu-ImageBasedGroupUpgrade.adoc[]
+--
+
+. Apply the created file by running the following command on the hub cluster:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+.Verification
+
+* Monitor the status updates by running the following command:
++
+--
+[source,terminal]
+----
+$ oc get ibgu -o yaml
+----
+
```

Comment:
I noticed there is a lot of `--` open block usage throughout the PR when I think just about all of them could be just using `+` continuations instead (which is what guidelines usually show using)?
```suggestion
+
```
Is there a reason the open blocks throughout the PR couldn't be switched to `+`s? Didn't see any complex steps but I could be missing them / misunderstanding.